features/support/hooks.ts

```diff
@@ -37,6 +37,7 @@ Before(function(
 
   const tmpDirNodeModulesPath = path.join(this.tmpDir, 'node_modules')
   const tmpDirCucumberPath = path.join(tmpDirNodeModulesPath, 'cucumber')
+  // @ts-ignore
```

Comment:
What is being ignored here?

Reply:
`fs-extra` typings seem to be out of sync with the code we are using - this is one I'm looping back to once I get the rest it green

Reply:
This is a strange one - the `createSymlink` and `createSymlinkSync` methods have been on fs-extra [for years](https://github.com/jprichardson/node-fs-extra/blame/master/lib/ensure/index.js) but they aren't documented and aren't in the DefinitelyTyped definitions.

I'll make PRs for both, link them up here, and see what comes out of it.

Reply:
I've switched our code to use `createSymlinkSync` which has the same outcome, and raised https://github.com/jprichardson/node-fs-extra/issues/758 to query the API status of the other method.